gax/src/main/java/com/google/api/gax/rpc/CallableFactory.java

```diff
@@ -71,91 +65,26 @@ public static CallableFactory create(TransportDescriptor transportDescriptor) {
-   * @param pagedCallSettings {@link PagedCallSettings} to configure the paged settings with.
-   * @param clientContext {@link ClientContext} to use to connect to the service.
-   * @return {@link UnaryCallable} callable object.
-   */
-  public <RequestT, ResponseT, PagedListResponseT>
-      UnaryCallable<RequestT, PagedListResponseT> createPagedVariant(
-          UnaryCallable<RequestT, ResponseT> transportCallable,
-          PagedCallSettings<RequestT, ResponseT, PagedListResponseT> pagedCallSettings,
-          ClientContext clientContext) {
-    UnaryCallable<RequestT, ResponseT> unaryCallable =
-        createBaseCallable(transportCallable, pagedCallSettings, clientContext);
-    UnaryCallable<RequestT, PagedListResponseT> pagedCallable =
-        new PagedCallable<>(unaryCallable, pagedCallSettings.getPagedListResponseFactory());
-    return new EntryPointUnaryCallable<>(
-        pagedCallable,
-        transportDescriptor.createDefaultCallContext(),
-        getCallContextEnhancers(clientContext));
-  }
-
-  /**
-   * Create a callable object that represents a simple call to a paged API method. Designed for use
-   * by generated code.
-   *
-   * @param transportCallable the callable that directly issues the call to the underlying API
-   * @param pagedCallSettings {@link PagedCallSettings} to configure the method-level settings with.
-   * @param clientContext {@link ClientContext} to use to connect to the service.
-   * @return {@link UnaryCallable} callable object.
-   */
-  public <RequestT, ResponseT, PagedListResponseT> UnaryCallable<RequestT, ResponseT> create(
-      UnaryCallable<RequestT, ResponseT> transportCallable,
-      PagedCallSettings<RequestT, ResponseT, PagedListResponseT> pagedCallSettings,
-      ClientContext clientContext) {
-    UnaryCallable<RequestT, ResponseT> unaryCallable =
-        createBaseCallable(transportCallable, pagedCallSettings, clientContext);
-    return new EntryPointUnaryCallable<>(
-        unaryCallable,
-        transportDescriptor.createDefaultCallContext(),
-        getCallContextEnhancers(clientContext));
+    return new RetryingCallable<>(
+        clientContext.getDefaultCallContext(), innerCallable, retryingExecutor);
   }
 
   /**
    * Create a callable object that represents a batching API method. Designed for use by generated
    * code.
    *
-   * @param transportCallable the callable that directly issues the call to the underlying API
+   * @param innerCallable the callable to issue calls
    * @param batchingCallSettings {@link BatchingSettings} to configure the batching related settings
    *     with.
    * @param context {@link ClientContext} to use to connect to the service.
    * @return {@link UnaryCallable} callable object.
    */
-  public <RequestT, ResponseT> UnaryCallable<RequestT, ResponseT> create(
-      UnaryCallable<RequestT, ResponseT> transportCallable,
+  @BetaApi
+  public <RequestT, ResponseT> UnaryCallable<RequestT, ResponseT> withBatching(
```

Comment:
the `withSomething` naming pattern is usually used if class creates a new copy of itself as returned value:
```java
class T {
    public T withSometihng(someOtherStuff) {
         //...
         return new T(this, plusSomeOtherStuff);
    }
}
```
here it is in a factory class (please read the next review comment on L136).